### PR TITLE
Fix segmentation fault when running backup against collective with insecure certificate

### PR DESCRIPTION
**Pull request by mandopaloooza**
sdpctl hits seg fault when we check if the response status contains Forbidden. Check if response is nil before checking the status code.

```
$ sdpctl appliance backup
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x100746cfa]

goroutine 1 [running]:
github.com/appgate/sdpctl/pkg/appliance.backupEnabled({0x100aeb7b8, 0x101167840}, 0xc00027a000, {0xc000280000, 0x142b}, 0x0)
	github.com/appgate/sdpctl/pkg/appliance/backup.go:431 +0xba
github.com/appgate/sdpctl/pkg/appliance.PerformBackup(0x0?, {0x101167840, 0x0, 0x0?}, 0xc000236080)
	github.com/appgate/sdpctl/pkg/appliance/backup.go:94 +0x245
...
```

### CODE REVIEW

**Comment by mandopaloooza** [comment]
Error is properly handled now
```
2 errors occurred:
        * Failed to determine backup option: No response Get "https://envy-10-98-137-2.devops:8443/admin/global-settings": tls: failed to verify certificate: x509: certificate signed by unknown authority
        * Trust the certificate or import a PEM file using 'sdpctl configure --pem=<path/to/pem>'
```

**Review by userappgate — APPROVED** [review]

File: go.sum [hunk 1]
```diff
@@ -438,6 +438,7 @@ golang.org/x/sys v0.0.0-20220722155257-8c9f86f7a55f/go.mod h1:oPkhp1MJrh7nUepCBc
 golang.org/x/sys v0.0.0-20220811171246-fbc7d0a398ab/go.mod h1:oPkhp1MJrh7nUepCBck5+mAzfO9JrbApNNgaTdGDITg=
 golang.org/x/sys v0.0.0-20220908164124-27713097b956/go.mod h1:oPkhp1MJrh7nUepCBck5+mAzfO9JrbApNNgaTdGDITg=
 golang.org/x/sys v0.6.0/go.mod h1:oPkhp1MJrh7nUepCBck5+mAzfO9JrbApNNgaTdGDITg=
+golang.org/x/sys v0.13.0/go.mod h1:oPkhp1MJrh7nUepCBck5+mAzfO9JrbApNNgaTdGDITg=
```
Comment (kajes) [inline go.sum:441]: This diff does not seem related to the fix. Guessing it slipped in somehow?

**Review by kajes — APPROVED** [review]
One comment, otherwise LGTM.

File: go.sum (hunk 1 above)
Comment (mandopaloooza) [reply go.sum:441]: I'll revert
